index.html

```diff
@@ -296,9 +289,6 @@ <h2>
           ></textarea>
         </fieldset>
         <input type="submit" id="submit" name="submit" value="Get in touch" />
-        <small class="small hidden show"
-          >Please, enter valid Email: lower case only.</small
-        >
       </form>
     </section>
     <!--Footer-->
```

Comment:
- [ OPTIONAL] Let's increase the weight of this vertical footer to line up your social media icons to make it more visible

- Figma design:

![good3](https://user-images.githubusercontent.com/79058364/161404876-79a0ceab-ccb9-46c7-a013-2870ce4878ad.png)


- Your design:

![bad3](https://user-images.githubusercontent.com/79058364/161404885-1630a1c8-7db1-44a3-9a98-72156a62219d.png)